packages/react-storage/src/components/StorageImage/StorageImage.tsx

```diff
@@ -14,11 +14,12 @@ export const StorageImage = ({
   identityId,
   imgKey,
   onStorageGetError,
+  validateObjectExistence = true,
```

Comment:
What if the provided value is `false` or `null` 😅

Reply:
It works? I just tried it. Or do you mean should we change this boolean prop to be the opposite so the default is false?

Reply:
Yeah, if they provide `false` or `null`, I think it would default to `true` right?

Reply:
> Yeah, if they provide false or null, I think it would default to true right

This 🤠 Maybe just add something like:
```
const resolvedValidateObjectExistence = isUndefined(validateObjectExistence) ? true : validateObjectExistence;
```

Reply:
ugh fine. a bit wordy, but changed

Reply:
Mean you could've renamed it 😏